docs/4.3/quickstart.md

```diff
@@ -274,6 +279,91 @@ $ echo "Awesome!"
 # check out your shared ssh session between two CLI windows
 ```
 
+## Run Teleport using Docker
+
+We provide pre-built Docker images for every version of Teleport. These images are hosted on quay.io.
+
+- [All tags under `quay.io/gravitational/teleport` are Teleport Community images](https://quay.io/repository/gravitational/teleport?tag=latest&tab=tags)
+
+We currently only offer Docker images for `x86_64` architectures.
+
+### Pick your image
+
+This table gives an idea of how our image naming scheme works. We offer images which point to a static version of Teleport, as well as images which are
+automatically rebuilt every night. These nightly images point to the latest version of Teleport from the current release branch, plus the two previous release branches.
```

Comment:
Copy the sentence from enterprise version.
Maybe the whole thing should link here? And the enterprise version only explain the image selection + license file?

Reply:
Unfortunately the `tctl users add` syntax is different for OSS versus Enterprise, plus I had to do some extra fudging to get the license file to work for Enterprise. Painful as it is, it's probably best to keep them separate.